docs/content/configuration/global-configuration/configmap-resource.md

```diff
@@ -2,16 +2,16 @@
 docs: DOCS-586
 doctypes:
 - ''
-title: ConfigMap Resources
+title: ConfigMap resources
 toc: true
-weight: 1600
+weight: 300
 ---
 
-The ConfigMap Resources allows you to customize or fine tune NGINX behavior. Examples include setting the number of worker processes or customizing the access log format.
+When using F5 NGINX Ingress Controller, you can customize or fine tune NGINX behaviour using ConfigMap resources. Examples include setting the number of worker processes or customizing the access log format.
```

Comment:
> customize (US)
> behaviour (UK)
Is there a preferred style between US and UK English for our docs, or are both acceptable?

Reply:
Good catch, we're meant to stick to US spelling. Updating it in a moment.